src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.targets

```diff
@@ -91,10 +91,34 @@
     </ItemGroup>
   </Target>
 
+   <Target Name="_RazorCreateSourceHashFiles" DependsOnTargets="_RazorResolveSourceFiles">
+    <PropertyGroup>
+      <RazorSourcesHashFile>$(IntermediateOutputPath)$(MSBuildProjectName).RazorSourceHash.cache</RazorSourcesHashFile>
```

Comment:
underscore this guy yeah?

Reply:
I would think so too. We might need to review the embedded files thing too - https://github.com/aspnet/FileSystem/blob/dev/src/FS.Embedded/build/netstandard2.0/Microsoft.Extensions.FileProviders.Embedded.targets#L27. The pattern \ casing has been copied from the sdk - https://github.com/dotnet/sdk/blob/2995c8495082d1dd4bcad35c37007375edf45c6f/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.GenerateAssemblyInfo.targets#L99